Project_FreeRTOS/main/task_rung_output.h

```diff
@@ -13,37 +13,46 @@ enum BuzzerStatus
 };
 
 // Hàm điều khiển buzzer mức cảnh báo thấp
-void buzzerAlertLow()
+void buzzerAlertLow(void *pvParameters)
 {
-  Serial.println("Buzzer: Low alert!");
-  for (int i = 0; i < 2; i++) // Rung 2 lần
+  Serial.println("Buzzer Low Alert Task Started.");
+  // Thực hiện cảnh báo mức thấp
+  for (int i = 0; i < 4; i++) // báo 4 lần
   {
     analogWrite(buzzer_pin, 200); // Tín hiệu PWM (50% duty cycle)
-    delay(200);                  // Rung trong 200ms
+    vTaskDelay(200 / portTICK_PERIOD_MS);  // báo trong 200ms
     analogWrite(buzzer_pin, 0);  // Tắt tín hiệu
-    delay(800);                  // Nghỉ 800ms
+    vTaskDelay(20 / portTICK_PERIOD_MS);  // Nghỉ 20ms
   }
+  Serial.println("Buzzer Low Alert Task Completed.");
+  // Kết thúc sau khi hoàn thành
+  vTaskDelete(NULL);
 }
 
 // Hàm điều khiển buzzer mức cảnh báo cao
-void buzzerAlertHigh()
+void buzzerAlertHigh(void *pvParameters)
 {
-  Serial.println("Buzzer: High alert!");
-  for (int i = 0; i < 8; i++) // Rung 8 lần
+  Serial.println("Buzzer High Alert Task Started.");
+  // Thực hiện cảnh báo mức cao
+  for (int i = 0; i < 8; i++) // báo 8 lần
```

Comment:
Should update for loop to unfinity loop, we need the buzzer continue to ring whatever received new message(new pattern)